src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/AllocCoTaskMemTests.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class AllocCoTaskMemTests
+    {
+        [Theory]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(100)]
+        public void AllocCoTaskMem_Int32_ReadableWritable(int size)
```

Comment:
This test isn't needed. Most of the tests under `src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/Copy/` depend on this functionality and do this exact scenario.

Reply:
I say let's keep them - they already existed in `MarshalTests.cs` before this PR. They don't really test much but its good to be sure of the coverage and I don't see a point in removing more specific exisiting tests

Reply:
>They don't really test much but its good to be sure of the coverage and I don't see a point in removing more specific exisiting tests

The point is not having duplicate validation of the same thing. Even by your own admission they don't "test much" - since that is the case, remove them. We shouldn't have tests just for the sake of having a test. If it exists it better indicate something real and in this case it won't because literally half of the tests in for the Copy API will fail.